cogs/remind_me.py

```diff
@@ -154,21 +158,30 @@ async def autocomplete_get_delays(ctx: TeXBotAutocompleteContext) -> set[str]:
                     if month < 10:
                         delay_choices.add(f"0{month}{joiner}{year}")
 
-        elif match := re.match(r"\A(?P<date>\d{1,2}) ?[/\-.] ?(?P<month>\d{1,2})\Z", ctx.value):  # noqa: E501
+        elif match := re.match(
+            r"\A(?P<date>\d{1,2}) ?[/\-.] ?(?P<month>\d{1,2})\Z",
+            ctx.value,
+        ):
             if 1 <= int(match.group("date")) <= 31 and 1 <= int(match.group("month")) <= 12:
                 for year in range(current_year, current_year + 40):
                     for joiner in ("/", " / ", "-", " - ", ".", " . "):
                         delay_choices.add(f"{joiner}{year}")
 
-        elif match := re.match(r"\A(?P<date>\d{1,2}) ?[/\-.] ?(?P<month>\d{1,2}) ?[/\-.] ?\Z", ctx.value):  # noqa: E501
+        elif match := re.match(
+            r"\A(?P<date>\d{1,2}) ?[/\-.] ?(?P<month>\d{1,2}) ?[/\-.] ?\Z",
+            ctx.value,
+        ):
             if 1 <= int(match.group("date")) <= 31 and 1 <= int(match.group("month")) <= 12:
                 for year in range(current_year, current_year + 40):
                     delay_choices.add(f"{year}")
 
-        elif match := re.match(r"\A(?P<date>\d{1,2}) ?[/\-.] ?(?P<month>\d{1,2}) ?[/\-.] ?(?P<partial_year>\d{1,3})\Z", ctx.value):  # noqa: E501
+        elif match := re.match(
+            r"\A(?P<date>\d{1,2}) ?[/\-.] ?(?P<month>\d{1,2}) ?[/\-.] ?(?P<partial_year>\d{1,3})\Z",  # noqa: E501
+            ctx.value,
+        ):
             if 1 <= int(match.group("date")) <= 31 and 1 <= int(match.group("month")) <= 12:
                 for year in range(current_year, current_year + 40):
-                    delay_choices.add(f"{year}"[len(match.group("partial_year")):])
+                    delay_choices.add(f"{year}"[len(match.group("partial_year")) :])
```

Comment:
Spaces should not be included around slice indexing syntax. See [this StackOverflow answer](https://stackoverflow.com/a/13842622/14403974) for more details.